test/it/surveyTests/nodeDefTest.js

```diff
@@ -12,14 +12,14 @@ const fetchRootNodeDef = async () => {
   return await NodeDefRepository.fetchRootNodeDef(Survey.getId(survey), true)
 }
 
-const createNodeDef = (parentNodeUuid, type, name) =>
-  NodeDef.newNodeDef(parentNodeUuid, type, Survey.cycleOneKey, {
+const createNodeDef = (parentNode, type, name) =>
```

Comment:
Can you rename parentNode to nodeDefParent to be consistent with our convention?

Reply:
Ok